book/src/become-a-validator-docker.md

```diff
@@ -0,0 +1,69 @@
+# Become a Validator: Using Docker
+
+Sigma Prime maintains the
+[sigp/lighthouse-docker](https://github.com/sigp/lighthouse-docker) repository
+which provides an easy way to run Lighthouse without building the Lighthouse
+binary yourself.
+
+> Note: when you're running the Docker Hub image you're relying upon a
+> pre-built binary instead of building from source. If you want the highest
+> assurance you're running the _real_ Lighthouse,
+> [build the docker image yourself](./docker.md) instead. You'll need some
+> experience with docker-compose to integrate your locally built docker image
+> with the docker-compose environment.
+
+### 1. Clone the repository
+
+Once you have docker-compose
+[installed](https://docs.docker.com/compose/install/), clone the
+[sigp/lighthouse-docker](https://github.com/sigp/lighthouse-docker) repository.
+
+```bash
+$ git clone https://github.com/sigp/lighthouse-docker
+$ cd lighthouse-docker
+```
+
+### 2. Configure the docker environment
+
+Then, create a `.env` file with the following contents (these values are
```

Comment:
I would suggest being more explicit (had a misunderstanding while following the guide):
`Then, create a file named `.env` with the following contents  ...`